src/Websites/Websites/Cmdlets/WebApps/PublishAzureWebApp.cs

```diff
@@ -74,6 +74,8 @@ public override void ExecuteCmdlet()
 
             Action zipDeployAction = () =>
             {
+                if (!Path.IsPathRooted(ArchivePath))
+                    ArchivePath = (this.SessionState.Path.CurrentFileSystemLocation).Path + ArchivePath;
```

Comment:
```suggestion
                    ArchivePath = Path.Join(this.SessionState.Path.CurrentFileSystemLocation).Path, ArchivePath);
```

Reply:
I believe this will hand ../Sample/*.Zip as well.

Reply:
`Path.Join()` is not available in .NetStandard.

Reply:
Alternatively, we can use Path.Combine()